community-operators/okd-pipeline-operator/1.2.3/manifests/openshift-pipelines-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,363 @@
+--- 
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata: 
+  annotations: 
+    support: OKD working group
+    alm-examples: |-
+        [
+          {
+            "apiVersion": "operator.tekton.dev/v1alpha1",
+            "kind": "Config",
+            "metadata": {
+              "name": "name.must.be-cluster"
+            },
+            "spec": {
+              "targetNamespace": "openshift-pipelines"
+            }
+          }
+        ]
+    capabilities: "Seamless Upgrades"
+    containerImage: quay.io/jomeier/tektoncd-pipeline-operator
+    description: >-
+        OKD Pipelines is a cloud-native CI/CD solution for building
+        pipelines using Tekton concepts which run natively on OpenShift, Kubernetes and OKD.
+    createdAt: '2021-01-31T18:16:25Z'
+    repository: 'https://github.com/jomeier/tektoncd-pipeline-operator/tree/okd-v1.2.3'
+  name: okd-pipelines-operator.v1.2.3
+  namespace: placeholder
+spec: 
+  apiservicedefinitions: {}
+  customresourcedefinitions: 
+    owned: 
+      - description: "Config is the Schema for the configs API"
+        kind: Config
+        name: config.operator.tekton.dev
+        version: v1alpha1
+  description: |
+      OKD Pipelines is a cloud-native continuous integration and delivery
+      (CI/CD) solution for building pipelines using [Tekton](https://tekton.dev).
+      Tekton is a flexible Kubernetes-native open-source CI/CD framework, which
+      enables automating deployments across multiple platforms (Kubernetes,
+      serverless, VMs, etc) by abstracting away the underlying details.
+      
+      ## Features
+      * Standard CI/CD pipelines definition
+      * Build images with Kubernetes tools such as S2I, Buildah, Buildpacks, Kaniko, etc
+      * Deploy applications to multiple platforms such as Kubernetes, serverless and VMs
+      * Easy to extend and integrate with existing tools
+      * Scale pipelines on-demand
+      * Portable across any Kubernetes platform
+      * Designed for microservices and decentralized team
+      * Integrated with OpenShift Developer Console
+      
+      ## Installation
+      _OKD Pipelines Operator_ gets installed into a single namespace (openshift-operators) which would then install _Red Hat OpenShift Pipelines_ into the openshift-pipelines namespace. _Red Hat OpenShift Pipelines_ is however cluster-wide and can run pipelines created in any namespace.
```

Comment:
Given that OpenShift Pipelines is a product, `OKD Pipelines` doesn't feel right to me in the context of community operators :)